aries-site/src/data/wcag/components.json

```
@@ -862,5 +862,519 @@
+        "label": "Images of Text",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.6",
+        "label": "Contrast (Enhanced)",
+        "status": "failed",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.7",
+        "label": "Low or No Background Audio",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.8",
+        "label": "Visual Presentation",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.9",
+        "label": "Images of Text (No Exception)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.10",
+        "label": "Reflow",
+        "status": "failed",
+        "notes": "Unable to access calendar when scrolling on small viewport"
+      },
+      {
+        "rule": "1.4.11",
+        "label": "Non-text Contrast",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.12",
+        "label": "Text Spacing",
+        "status": "failed",
+        "notes": ""
+      },
+      {
+        "rule": "1.4.13",
+        "label": "Content on Hover or Focus",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.1.1",
+        "label": "Keyboard",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "2.1.2",
+        "label": "No Keyboard Trap",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "2.1.3",
+        "label": "Keyboard (No Exception)",
+        "status": "failed",
+        "notes": "When calendar closes, the focus doesn't return to the input."
+      },
+      {
+        "rule": "2.2.1",
+        "label": "Timing Adjustable",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.2.2",
+        "label": "Pause, Stop, Hide",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.2.3",
+        "label": "No Timing",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.2.4",
+        "label": "Interruptions",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.2.5",
+        "label": "Re-authenticating",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.2.6",
+        "label": "Timeouts",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.3.1",
+        "label": "Three Flashes or Below Threshold",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.3.2",
+        "label": "Three Flashes",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "2.3.3",
+        "label": "Animation from Interactions",
+        "status": "not-applicable",
```

```suggestion
        "status": "conditional",
```
I missed this rule when reviewing the xcel spreadsheet but the animation that is triggered when the calendar changes month can be turned off with a prop in grommet, I think we should mark this as conditional